apps/events/migrations/0011_no_activity_pings_to_events_20240619_1436.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.2.11 on 2024-06-19 14:36
+
+from django.db import migrations
+
+from apps.events.models import EventActionType
+
+
+def convert_no_activity_pings_to_event(apps, schema_editor):
+    NoActivityMessageConfig = apps.get_model("experiments", "NoActivityMessageConfig")
+    TimeoutTrigger = apps.get_model("events", "TimeoutTrigger")
+    EventAction = apps.get_model("events", "EventAction")
+
+    for no_activity_ping in NoActivityMessageConfig.objects.all():
+        for experiment in no_activity_ping.experiment_set.all():
+            action = EventAction.objects.create(
+                action_type=EventActionType.SEND_MESSAGE_TO_BOT,
+                params={"message_to_bot": no_activity_ping.message_for_bot}
+            )
+            TimeoutTrigger.objects.create(
```

Comment:
This [will cause](https://github.com/dimagi/open-chat-studio/blob/33ba3ec0d6b53d5b301647317fc19a9e2557e6bb/apps/events/models.py#L212) all newly created triggers to go off since there are no event logs for these triggers?

Reply:
Yikes, great point! https://github.com/dimagi/open-chat-studio/pull/468/commits/b7ab9ae8d32244b4a09f4cbce4261c6325af8001

Reply:
I'm actually not sure my solution is totally correct actually. I think I need to attach the EventLog to the last human message so that it still won't trigger. I'll look again tomorrow.

Reply:
OK, added that here: https://github.com/dimagi/open-chat-studio/pull/468/commits/36ab331edf9b45d966dc8dc7f40cceb6fa46b689